src/pages/[platform]/build-a-backend/add-aws-services/interactions/chatbot/index.mdx

```diff
@@ -0,0 +1,68 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Interact with bots',
+  description: 'Learn more about how to integrate chat bot interactions into your application using Amplify.',
+  platforms: [
+    'javascript',
+    'react-native',
+    'angular',
+    'nextjs',
+    'react',
+    'vue'
+  ],
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+
+### Send messages to bot
+You can send a text message to chatbot backend with `send()` command. The method returns a promise that includes the chatbot response.
+
+```javascript
```

Comment:
can we add sample filenames to the examples to show that this is a client-side API? e.g. `src/App.tsx` or `src/my-client-side.js`